docs/index.md

```diff
@@ -46,8 +51,34 @@ Hello World
 => true
 ```
 
-As you can see from log, every action execution is wrapped inside a DB transaction.
-The main difference between these methods is: `#perform!` raises exception in case of errors and `#perform` simply returns `false`
+You can use `Granite::Action.transaction` explicitly and wrap any logic in transaction:
+
+```ruby
+Granite::Action.transaction do
+  some_other_logic
+  Action.new.perform!
+end
+```
+
+You can use `after_commit` callback as well.
```

Comment:
This should be in a different section `### Callbacks`, don't you think?

Reply:
Also, it would be good to have an explanation about when the `after_commit` is called, with maybe an example with nested transactions, and in which cases the block won't be called.